samples_project/Assets/Editor/EditorAssemblyDef.asmdef

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "NewAssemblyDef",
```

Comment:
This line needs to be updated to reflect the name of the file

Reply:
hm, I had changed it but must not have saved, fixed